queries/v/highlights.scm

```diff
@@ -288,7 +288,7 @@
   (block_comment)
 ] @comment @spell
 
-(_
```

Comment:
I don't think this is the correct fix. doc comments are comments that is followed by a func/type/enum right?

The fix should be
```diff
(_
  (line_comment)+ @comment.documentation
+ .
  [
    (function_declaration)
    (type_declaration)
    (enum_declaration)
  ])
```

Reply:
A different fix:
```diff
(_
+ (line_comment) @comment.documentation
+ .
  (line_comment)* @comment.documentation
+ .
  [
    (function_declaration)
    (type_declaration)
    (enum_declaration)
  ])
```

Reply:
We can start thinking about dropping 0.9 compatibility, although I usually prefer to wait for a 0.x.1  that has ironed out most of the rough bits of a major (zerover) release.